incubator/hnc/internal/pkg/selectors/selectors.go

```diff
@@ -67,3 +76,24 @@ func getSelectorFromString(str string) (labels.Selector, error) {
 	}
 	return selector, nil
 }
+
+// GetNoneSelector returns true indicates that user do not want this object to be propagated
+func GetNoneSelector(inst *unstructured.Unstructured) (bool, error) {
+	noneSelectorStr := GetNoneSelectorAnnotation(inst)
+	// Empty string is treated as 'false'. In other selector cases, the empty string is auto converted to
+	// a selector that matches everything.
+	if noneSelectorStr == "" {
+		return false, nil
+	}
+	noneSelector, err := strconv.ParseBool(noneSelectorStr)
+	if err != nil {
+		// Returning false here in accordance to the Go standard
+		return false,
+			fmt.Errorf("invalid %s value, %w",
```

Comment:
Please use a colon instead of a comma - wrapped errors should ~always be after a colon.

`invalid %s value: %w`

Reply:
done